src/services/offerService.js

```diff
@@ -325,12 +347,16 @@ export const editOffer = measureTime(TIMED_ACTIONS.OFFER_EDIT, async ({
         return json;
 
     } catch (error) {
-        if (!isErrorRegistered) createEvent(EVENT_TYPES.ERROR(
+
+        const errorArray = Array.isArray(error) ? error :
+            [{ msg: "Unexpected Error. Please try again later." }];
+
+        if (!isErrorRegistered) createErrorEvent(
```

Comment:
when the function call spans over multiple lines, it's better to have the `{}` around the if body